### PR TITLE
Update catalog deployment swagger to 8.4.1

### PR DESCRIPTION
**Pull request by frodenas**
Update catalog deployment swagger to 8.4.1.

Note: the swagger definition has started including a `security` section for each operation. The [fix_catalog_deployment_swagger](https://github.com/vmware/vra-sdk-go/blob/master/hack/fix_catalog_deployment_swagger) script now includes an option to remove this section.

Signed-off-by: Ferran Rodenas <rodenasf@vmware.com>

### CODE REVIEW

File: hack/fix_catalog_deployment_swagger [hunk 1]
```diff
@@ -56,8 +56,22 @@ def rename_deployment_models(swagger):
         replace_value(swagger, "$ref", oldRef, newRef)
 
 
+def remove_global_security(swagger):
```
Comment (Prativa20) [inline hack/fix_catalog_deployment_swagger:59]: Can we add some detail about why are we adding this hack and when can this be removed? Is there any bug/issue that is opened for the API and all such information.

**Review by Prativa20 — COMMENTED** [review]
Minor comment

File: hack/fix_catalog_deployment_swagger (hunk 1 above)
Comment (frodenas) [reply hack/fix_catalog_deployment_swagger:59]: Added a comment at https://github.com/vmware/vra-sdk-go/blob/29a153c07e336ee1dd41699a346c0308e73828ff/hack/fix_catalog_deployment_swagger#L64

**Review by Prativa20 — APPROVED** [review]
Looks good!

**Pull request merged**